packages/paste-core/components/meter/src/MeterLabel.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {type BoxProps, Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import type {HTMLPasteProps} from '@twilio-paste/types';
+
+import {labelIdSuffix} from './Meter';
+
+export interface MeterLabelProps extends HTMLPasteProps<'div'>, Pick<BoxProps, 'element'> {
+  children: string;
+  htmlFor: string;
+}
+
+const MeterLabel = React.forwardRef<HTMLLabelElement, MeterLabelProps>(
+  ({element = 'METER_LABEL', children, htmlFor, ...labelProps}, ref) => {
+    return (
+      <Box display="flex" width="fit-content" element={`${element}_WRAPPER`}>
+        <Label {...labelProps} as="div" element={element} id={`${htmlFor}${labelIdSuffix}`} ref={ref}>
```

Comment:
suggestion: I suspect we need some tests around ensuring our slight of hand remains working when we do our labelling, both here and in the meter itself

Reply:
added some more but let me know if there's more coverage you want to see